### ISSUE TITLE
Reusable component?

### ISSUE THREAD

**Issue by testerez** [jslauthor/react-bezier-component#2]
Wow, looks super nice! Do you plan to bundle it as a reusable component?

**Comment by jslauthor** [comment]
@testerez No, plans, currently. 😬 It's more of an example for others to work from.
